Update fail conditions: danger zone + exit interview

tannerwelsh opened the pull request:
Focus fail conditions on 3 stats: team play, cult. contrib., and XP
growth

Provide some notes on why fail conditions are this way

tannerwelsh committed:
Update fail conditions: danger zone + exit interview

Focus fail conditions on 3 stats: team play, cult. contrib., and XP
growth

Provide some notes on why fail conditions are this way

tannerwelsh commented:
Based on convo w/ @shereefb today; fixes #49.

Please read and give feedback @shereefb @needdra @carlabagdonas

carlabagdonas commented on Game_Manual/Boundaries.md:
```diff
@@ -19,18 +19,29 @@ At this point, they are ready to pursue their craft as a practitioner and no lon
+
+The second stage is an _Exit Interview_, and is entered into when a player's stats have _both_ (a) been in the Danger Zone for at least two weeks and (b) have not shown any improvement.
+
+During the Exit Interview, the player will meet with a moderator to discuss their performance and whether or not they can continue in the game. If the moderator is not convinced that the player is putting in the right effort, they will ask them to leave the game.
```
I'm not sure about using "right effort" as the guiding principle here. In some cases, I think someone can be putting in as much effort as possible, and just not have the skills necessary to succeed in the environment.

Commit:
Small edits 

Removed the **right effort** statement about how a moderator decides wether or not someone will leave.

carlabagdonas commented on Game_Manual/Boundaries.md:
```diff
@@ -19,18 +19,29 @@ At this point, they are ready to pursue their craft as a practitioner and no lon
+- [Team Play][team-play] : 80
+- [Culture Contribution][culture-contribution] : 80
+- [XP][xp] earned per cycle : 15
 
```
Why isn't learning contribution included as a boundary stat?
